sim/rogue/poisons.go

```diff
@@ -322,6 +368,10 @@ func (rogue *Rogue) applyNumbingPoison() {
 				rogue.NumbingPoison[NormalProc].Cast(sim, result.Target)
 			}
 		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			fmt.Println("Numbing Poison deactivated")
+			rogue.PoisonsActive--
+		},
 	})
```

Comment:
What you're looking for are the functions `func (rogue *Rogue) makeXPoison(procSource ProcPoisonSource)`. Those are the ones actually building the spells that act as the poisons / debuff applicators

Reply:
For example, here is where you probably want to add `OnGain` and `OnExpire` to increment for Occult
![image](https://github.com/user-attachments/assets/f77fa963-4be4-4c78-bec5-0d915390d0da)